src/agent/src/cdh.rs

```diff
@@ -90,6 +80,56 @@ impl CDHClient {
     }
 }
 
+/// Init the cdh client
+pub async fn init_cdh_client() -> Result<()> {
+    let mut cdh_client = CDH_CLIENT.write().await;
+    if cdh_client.is_none() {
+        let client = CDHClient::new().context("Failed to create CDH Client")?;
+        *cdh_client = Some(client);
+    }
+    Ok(())
+}
+
+/// Check if the CDH client is initialized
+pub async fn is_cdh_client_initialized() -> bool {
+    let cdh_client = CDH_CLIENT.read().await;
+    cdh_client.is_some() // Returns true if CDH_CLIENT is initialized, false otherwise
```

Comment:
The function name is pretty clear, so we might not need the notes here

Reply:
Fixed.